care/facility/tests/test_bed_create.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+from rest_framework import status
+from care.facility.api.viewsets.bed import BedViewSet
+from care.facility.tests.mixins import TestClassMixin
+
+class SingleBedTest(TestClassMixin, TestCase):
+    def test_create(self):
+        user = self.users[0]
+        sample_data = {
+            "bed_type": "REGULAR",
+            "description": "Testing creation of beds.",
+            "facility": "657c32be-d584-476c-9ce2-0412f0e7692e",
+            "location": "7f0c2bba-face-47f1-a547-87617a8c9b04",
```

Comment:
avoid hardcoding these, refer the already created sel.facility and create a location in the function